src/unitxt/operator.py

```diff
@@ -185,10 +188,7 @@ def is_should_be_processed(self, stream_name):
         ) and stream_name not in self.dont_apply_to_streams
 
     def _process_stream(self, stream: Stream, stream_name: str = None) -> Generator:
-        if self.is_should_be_processed(stream_name):
-            yield from self.process(stream, stream_name)
-        else:
-            yield from stream
+        yield from self.process(stream, stream_name)
```

Comment:
why was the check of `is_should_be_processed` removed here?

Reply:
Because the check was moved _process_multi_stream, is calls this method.

The issue was that this method was overriden by derived classes, and that's why the functionally did not work in derived classes. The _process_multi_stream method is not overriden.